velox/exec/AggregateUtil.h

```diff
@@ -34,4 +38,21 @@ struct AggregateRegistrationResult {
   }
 };
 
+class AggregateUtil {
+ public:
+  static AggregateInfo toAggregateInfo(
```

Comment:
Please, document this method.

There are a lot of parameter. Would it make changing this method to take AggregationNode and return a list of all AggregateInfos?